Helper/AutomatedTesting.php

```diff
@@ -165,6 +214,53 @@ public function __construct(
         $this->stockRegistry = $stockRegistry;
     }
 
+    /**
+     * Retrieve an Order from the repository
+     *
+     * @return Order|null
+     */
+    protected function getPastOrder()
+    {
+        try {
+            $sortOrder = $this->sortOrderBuilder
+                ->setField('entity_id')
+                ->setDirection('DESC')
+                ->create();
+
+            $nonZeroDiscountTaxSearchCriteria = $this->searchCriteriaBuilder
+                ->addFilter('discount_amount', 0, 'lt')
```

Comment:
gt?

Reply:
a discount actually shows up as negative in sales_order table, akin to whats shown from the sales order view
<img width="545" alt="Screen Shot 2021-05-07 at 10 23 58 AM" src="https://user-images.githubusercontent.com/76493951/117464201-6dbe4d80-af1e-11eb-8745-d399fa602d61.png">

Reply:
huh interesting